src/one_dragon/base/screen/screen_loader.py

```
@@ -302,3 +426,56 @@ def update_current_screen_name(self, screen_name: str) -> None:
         """
         self.last_screen_name = self.current_screen_name
         self.current_screen_name = screen_name
+
+    # ---- Screen Scope 管理 ----
+    # 通过 ScreenInfo.app_id 自动区分全局/局部 screen：
```

Ruff (0.15.20)

[warning] Comment contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF003)